dev-tools/omdb/src/bin/omdb/db.rs

```diff
@@ -1636,11 +1636,21 @@ struct CrucibleDatasetRow {
     no_provision: bool,
 
     // zpool fields
-    control_plane_storage_buffer: i64,
-    pool_total_size: i64,
+    #[tabled(display_with = "option_impl_display")]
+    control_plane_storage_buffer: Option<i64>,
+    #[tabled(display_with = "option_impl_display")]
+    pool_total_size: Option<i64>,
 
     // computed fields
-    size_left: i128,
+    #[tabled(display_with = "option_impl_display")]
+    size_left: Option<i128>,
+}
+
+fn option_impl_display<T: std::fmt::Display>(t: &Option<T>) -> String {
+    match t {
+        Some(v) => format!("{v}"),
+        None => String::from("n/a"),
```

Comment:
Not trying to bikeshed and probably not worth holding up the PR, but I'm not sure about `n/a`. A zpool could be missing from inventory for multiple reasons, right? At least

* the disk has been expunged
* the sled-agent holding this zpool failed to report its inventory in this collection pass

which seem pretty different in terms of "what should we report here". Can we tell the difference between these at this point? (Fine to say "let's land this as-is for R14 then figure out what to do here next week".)

Reply:
In this little display function, no, it just knows that it's an Option. It may make sense to put an enum here like

```rust
enum Thingy {
  PresentInInventory(value),
  
  NotPresentInInventory,
  
  Expunged,
}
```

I'd say we could check for time_deleted (except for https://github.com/oxidecomputer/omicron/issues/7962) or the backing physical disk status, but I'm inclined to land this for R14 as-is.